### PR TITLE
New, Clean UI

### PR DESCRIPTION
**Pull request by topisani**
I recently switched from Maple to sage, using it every day at school, and i am very happy with it. However i, as a web guy, think the current notebook interface looks quite outdated.

however, the current newui branch is quite buggy (no 3dplot  or mathjax support amongst others), and i personally dont agree 100% with the style.  

Therefore i am currently creating a new UI rewamp. It is based on the newest master branch, and the top priority is usability - both feature wise and UI wise.  
This is not finished, and it will not be for at least a couple of weeks, but i thought i would place this PR here already, to see if it has any interest.

Concerning the Sagemathcloud interface and the idea of porting it to sagenb, i personally think the sagemathcloud UI, while having quite a few nice touches, it also has a few annoying quirks, and the main point of this one is to be very clean and simple, which is what I think fits best for a program like this.

### CODE REVIEW

**Comment by kcrisman** [comment]
You may be interested in #136 and/or #242 in this regard.

Also, a mathjax upgrade would probably be best served by implementing #258 because then upgrading is just http://trac.sagemath.org/ticket/19875 (well, it is anyway, since http://trac.sagemath.org/ticket/17288 already is in Sage).  So I recommend not including that here, just not to confuse things.

**Comment by topisani** [comment]
This looks very interesting, i will take a closer look at #136, as that was one of the things i thought i probably wouldnt be able to do myself. As to the NewUI, i might update that to the #136 theme system once again, but i would probably mostly focus on my own cleanUI. The main goal for me is to just finally get something implemented, as there has been quite a bit of talk of a new UI over the years, but it seems to all have died.

TL;DR:
I will take a look at incorporating #136 and definately #258, and then add my own ui rewrite as a theme

**Comment by topisani** [comment]
i dont have time to finnish this anymore

**Comment by kcrisman** [comment]
Okay.  But keep in mind we welcome useful additions - you may also want to look at the Jupyter and SageMathCloud interfaces for other notebooks to contribute to.
